src/modules/roc_packet/fec.h

```diff
@@ -21,11 +21,11 @@ namespace packet {
 
 //! FECFRAME packet.
 struct FEC {
-    //! Seqnum of first source packet in block.
-    seqnum_t blknum;
+    //! The source block number.
+    blknum_t source_block_number;
 
-    //! The number of source packet per block.
-    size_t source_block_length;
+    //! The number of source packets per block.
+    blknum_t source_block_length;
```

Comment:
I'd clarify in the comment that it is the number of source packets in the block to which this packets belongs to. It's important since different blocks can have different sizes.

Reply:
Done